CHANGELOG.md

```diff
@@ -767,7 +767,7 @@ Version 2.0 of the UI framework is built for player 7.1. If absolutely necessary
 ## 1.0.0 (2017-02-03)
 - First release
 
-[3.32.0]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.32.0...HEAD
+[3.32.0]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.32.0...v3.33.0
```

Comment:
```suggestion
[3.33.0]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.32.0...v3.33.0
```

Reply:
maybe 🤔

Reply:
Problem was that it said `[3.32.0]` in the first place - this should have been `[develop]`

Reply:
But thanks for catching this!!! I kinda forgot this PR was in limbo. 👀